### PR TITLE
test: add testing for statistics of the container

### PR DESCRIPTION
**Pull request by chuanchang**
Add statistics of container testing.

### CODE REVIEW

**Comment by Starnop** [comment]
@chuanchang any update?

**Comment by chuanchang** [comment]
@Starnop please help review PR5 firstly, thanks.

File: pkg/validate/container.go [hunk 1]
```diff
@@ -142,6 +142,27 @@ var _ = framework.KubeDescribe("Container", func() {
 			cmd := []string{"not-exist-command"}
 			verifyExecSyncContainOutput(rc, containerID, cmd)
 		})
+
+		It("runtime should support ContainerStats getting expected error with non-existent container [Conformance]", func() {
+			By("test ContainerStats with non-existent container ID")
+			containerID := "foo"
+			expectError := true
```
Comment (Starnop) [inline pkg/validate/container.go]: not necessary?

**Comment by zhuangqh** [comment]
LGTM

**Pull request merged**